src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -17,7 +23,27 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+
+    private static final int MAX_LENGTH = 5;
+
+    public String getUniqueCharacters(String fileName) throws FileNotFoundException {
+        StringBuilder result = new StringBuilder();
+        Set<Character> charsFromFile = new TreeSet<>();
+        try (InputStream fileInputStream = new FileInputStream(fileName)) {
+            while (fileInputStream.available() > 0) {
+                char charReadedFromFile = (char) fileInputStream.read();
+                if (Character.isLetter(charReadedFromFile)) {
+                    charsFromFile.add(Character.toLowerCase(charReadedFromFile));
+                }
+            }
+        } catch (Exception e) {
+            throw new FileNotFoundException("File not Found!");
+        }
```

Comment:
you are throwing checked Exception from this method anyway and indicate this in method signature. try just removing this catch block and add IOException to method signature